inlong-sort/sort-connectors/doris/src/main/java/org/apache/inlong/sort/doris/table/DorisDynamicSchemaOutputFormat.java

```diff
@@ -123,18 +153,48 @@ public DorisDynamicSchemaOutputFormat(DorisOptions option,
             String tablePattern,
             boolean ignoreSingleTableErrors,
             String inlongMetric,
-            String auditHostAndPorts) {
+            String auditHostAndPorts,
+            boolean isSingle) {
```

Comment:
'isSingle' is not necessary to define, there is already a definition of multiple-sink？

Reply:
this is equivalent to !multipleSink, outputformat needs this to distinguish between the two different builders